sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -158,7 +158,7 @@ document_analysis_client = DocumentAnalysisClient(
 
 ### DocumentAnalysisClient
 `DocumentAnalysisClient` provides operations for analyzing input documents using prebuilt and custom models through the `begin_analyze_document` and `begin_analyze_document_from_url` APIs.
-Use the `model` parameter to select the type of model for analysis.
+Use the `model` parameter to select the type of model for analysis. The following table includes some of the models supported by the service (for a full list of supported models, see [here][fr-models]):
 
 |Model| Features
```

Comment:
wondering if it's better to just remove the table and the link out to the service docs. It's a nice visual, but I worry people might not read and miss the "includes some of the models". Just a thought, not blocking this PR

Reply:
Hmmm that's a better idea. I'll remove it now